README.md

```diff
@@ -39,7 +39,7 @@ For more information on configuring AppsFlyer as a destination in RudderStack an
 2. `RudderAppsFlyer` is available through [CocoaPods](https://cocoapods.org). To install it, add the following line to your `Podfile`:
 
 ```ruby
-pod 'RudderAppsFlyer', '~> 1.2.0'
+pod 'RudderAppsFlyer', '~> 1.2.1'
```

Comment:
It should be `1.3.0`, as the PR title is of type `feat`:
```suggestion
pod 'RudderAppsFlyer', '~> 1.3.0'
```
(If the PR is of type `fix` then we can update the patch version.)

Reply:
Done